### PR TITLE
update chinese translations

### PR DESCRIPTION
**Pull request by helight**
1.update some Chinese translations
2.do some cleanup

### CODE REVIEW

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 1]
```diff
@@ -2848,12 +2828,10 @@
         <source> You can <x id="START_LINK" ctype="x-a" equiv-text="&lt;a routerLink=&quot;deploy&quot;
          queryParamsHandling=&quot;preserve&quot;>"/>deploy a containerized app<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/>, select other namespace or <x id="START_LINK_1" equiv-text="&lt;a href=&quot;http://kubernetes.io/docs/user-guide/ui/&quot;
          target=&quot;_blank&quot;>"/>take the Dashboard Tour <x id="START_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;mat-icon class=&quot;kd-zerostate-icon&quot;>"/>open_in_new<x id="CLOSE_TAG_MAT_ICON" ctype="x-mat_icon" equiv-text="&lt;/mat-icon>"/><x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/> to learn more. </source>
-        <target>
-      你可以 <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>部署一个容器化应用<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/>, 选择其他 namespace，或者
+        <target>你可以 <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>部署一个容器化应用<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/>, 选择其他 namespace，或者
```
Comment (chenrui333) [inline i18n/zh-Hans/messages.zh-Hans.xlf:2831]:
```suggestion
        <target>你可以 <x id="START_LINK" ctype="x-a" equiv-text="&lt;a>"/>部署一个容器化应用<x id="CLOSE_LINK" ctype="x-a" equiv-text="&lt;/a>"/>, 选择其他命名空间，或者
```

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 2]
```diff
@@ -3018,161 +2996,143 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/frontend/chrome/nav/component.ts</context>
           <context context-type="linenumber">44</context>
         </context-group>
       </trans-unit>
       <trans-unit id="5deee9cd9fabadb49eb1e3fcae70ff560e41ba27" datatype="html">
         <source>Service Accounts </source>
-        <target>Service Accounts
-      </target>
+        <target>Service Accounts </target>
```
Comment (chenrui333) [inline i18n/zh-Hans/messages.zh-Hans.xlf:3103]:
```suggestion
        <target>服务账号</target>
```

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 3]
```diff
@@ -3763,19 +3694,15 @@
...
           <context context-type="sourcefile">src/app/frontend/create/from/form/createsecret/template.html</context>
           <context context-type="linenumber">74</context>
         </context-group>
       </trans-unit>
       <trans-unit id="a5e1deb7b16c5c6ddd4115d73bcd82bbf238dd99" datatype="html">
         <source> Data must be Base64 encoded. </source>
-        <target>
-              Data 必须是 Base64 编码的。
-            </target>
+        <target> Data 必须是 Base64 编码的。</target>
```
Comment (chenrui333) [inline i18n/zh-Hans/messages.zh-Hans.xlf:3705]:
```suggestion
        <target>数据必须是 Base64 编码的。</target>
```

File: i18n/zh-Hans/messages.zh-Hans.xlf [hunk 4]
```diff
@@ -3763,19 +3694,15 @@
       </trans-unit>
       <trans-unit id="77751725f4a7cb970bd5508543690c9e7977ab0d" datatype="html">
         <source> Data is required. </source>
-        <target>
-              Data 是必须的。
-            </target>
+        <target> Data 是必须的。</target>
```
Comment (chenrui333) [inline i18n/zh-Hans/messages.zh-Hans.xlf:3697]:
```suggestion
        <target> Data 是必需的。</target>
```

**Review by chenrui333 — APPROVED** [review]
minor improvement suggestions, others lgtm

**Review by chenrui333 — APPROVED** [review]
/lgtm
/approve

**Comment by helight** [comment]
@tanjunchen please take a look at this pr.

**Review by tanjunchen — COMMENTED** [review]
@helight   thanks for your job!
/lgtm

**Pull request merged**